.gitignore

```diff
@@ -5,4 +5,5 @@ docs/forge-server-utils/
 tmp/
 *.log
 .DS_Store
-Thumbs.db
\ No newline at end of file
+Thumbs.db
+package-lock.json
```

Comment:
Is there any specific reason for excluding the file? According to https://docs.npmjs.com/configuring-npm/package-lock-json.html:
> This file is intended to be committed into source repositories

Reply:
The only reason was to adhere with Augusto recommendation of us working with Autodesk npm registry, and not default registry.

Reply:
Ah ok, thanks. I think we'll need to revisit that recommendation.